src/main/java/org/testng/reporters/FileStringBuffer.java

```diff
@@ -28,6 +29,7 @@
 public class FileStringBuffer implements IBuffer {
   private static int MAX = 100000;
   private static final boolean VERBOSE = RuntimeBehavior.verboseMode();
```

Comment:
As you replaced System.out by a logger, I think VERBOSE can be removed now.
Instead, the logger should be configured according to the verboseMode value at the startup.

Reply:
I'm agree